wgpu-hal/src/vulkan/mod.rs

```diff
@@ -355,6 +355,13 @@ struct Swapchain {
     /// index as the image index, but we need to specify the semaphore as an argument
     /// to the acquire_next_image function which is what tells us which image to use.
     next_semaphore_index: usize,
+    /// The present timing information which will be set in the next call to [`present`](crate::Queue::present).
```

Comment:
```suggestion
    /// The present timing information which will be set in the next call to [`present()`](crate::Queue::present()).
```

Reply:
Thanks for pointing this out - I'm used to following the usual Rust conventions on this, but I see that wgpu has their own.